samples/i18n/src/components/TranslatableText/TranslatableText.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from "react";
+import { LayoutElement } from "@geocortex/web/components";
+import { UIContext } from "@geocortex/web/ui";
+import Typography from "@geocortex/web/ui/typography";
+
+export default function TranslatableText(props) {
+    const { translate } = useContext(UIContext);
+
+    return (
+        <LayoutElement {...props} style={{ backgroundColor: "white" }}>
+            <Typography variant="h3">Translate function:</Typography>
```

Comment:
It might be good to show another component other than `Typography` here (like `<Button>language-hello</Button>` or something), just to reinforce that all of our components handle translation. Otherwise someone might get the impression that they MUST use `Typography` or `translate()` to handle translation.

Reply:
Good call. I added a `Button` example and also added a note of this to the sample readme.